src/css/variables.scss

```diff
@@ -1,4 +1,4 @@
-$accent_color: #f9bb47;
+$accent_color: #0162AC;
 $background_color:  #1a353f;
 $shadow_color: #1a1b25;
```

Comment:
Great use of updating the variable color to make the app colors more accessible.

Reply:
These colors look great on the DOM and really match our current theme well. Great work!